Add note about shipyard repo for backports (#651)
submariner-io/submariner-website: src/content/development/backports/_index.en.md

`make LOCAL_BUILD=1 backport release=<release-branch> pr=<PR to cherry-pick>`

> Please ensure that [shipyard](https://github.com/submariner-io/shipyard) repo is checked out alongside the project.

Reviewer: Can you help me understand what problem this is trying to help people avoid? I tried a backport in the main repo without doing any special checkout of Shipyard and it worked fine. I do have both the main repo and Shipyard in my `~/go/src/submariner-io` dir, is that what you mean is required?

Author: Yes that's correct @dfarrell07. The shipyard repo should be alongside (i.e., $PWD/..) the current project.

> Please ensure that [shipyard](https://github.com/submariner-io/shipyard) repo is checked out alongside the project.

Reviewer: ```suggestion
Please ensure that [Shipyard's repo](https://github.com/submariner-io/shipyard) is checked out and updated alongside the project (`../<project dir where running make backport>`).
```

Thanks for clarifying @sridhargaddam! Maybe including this example in the docs would help them be more clear too?

Reviewer: ```suggestion
Since you are running with `LOCAL_BUILD=1`, ensure that [Shipyard's repo](https://github.com/submariner-io/shipyard) is checked out and updated alongside the project (`../<project dir where running make backport>`).
```

And this is the reasoning, correct? Might also be good to include.

`make LOCAL_BUILD=1 backport release=<release-branch> pr=<PR to cherry-pick>`

Since you are running with `LOCAL_BUILD=1`, ensure that [Shipyard's repo](https://github.com/submariner-io/shipyard) is

> checked out and updated alongside the project (`../<project dir where running make backport>`)

Reviewer: ```suggestion
checked out and updated alongside the project (`../<project dir where running make backport>`).
```
